Tests/test_PDB_MMCIFParser.py

```diff
@@ -81,8 +82,11 @@ def test_parsers(self):
             self.assertEqual(s.alphabet, generic_protein)
 
             # Here non-standard MSE are shown as M
-            self.assertEqual("MDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETLLVQ"
-                             "NANPDCKTILKALGPGATLEEMMTACQG", str(s))
+            self.assertEqual(
```

Comment:
These changes seem to reflect more formatting than the addition of the header code. Not an issue, but it'd be better to justify them than bundle them together in this PR.